spec/smpl_project_specific_spec.rb

```diff
@@ -76,18 +72,16 @@ def setup_dobj(druid)
       :label         => 'LabelQuux',
       :publish_attr  => { :publish => 'no', :shelve => 'no', :preserve => 'yes' },
       :project_style => {},
-      :bundle_dir    => @bundle_dir,
-      :smpl_manifest => @smpl_manifest,
+      :bundle_dir    => bundle_dir,
+      :smpl_manifest => smpl_manifest,
```

Comment:
why not pull `smpl_manifest` up to the same level as `bundle_dir` so as to not have to pass it in?  mostly just curious, wouldn't hold up merging over it.

Reply:
There are still some calls to this method at the example level (`it` block), below the level where a `let` block can be defined.